content/TrueCommand/Clusters.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Clusters Screen"
+description: "Descriptions of the options contained inside the TrueCommand Clusters menu."
+weight: 60
+geekdocCollapseSection: true
+tags:
+ - scaleclustering
+aliases:
+ - /truecommand/clustering/
+---
+
+{{< toc >}}
+
+TrueCommand 2.2, in conjunction with TrueNAS SCALE, can create and manage clusters, cluster volumes, and cluster volume shares.
+
+{{< hint danger >}}
+Clusters are an experimental feature in TrueCommand 2.2.
+Before using such features, please back up all your data.
+Do not rely on this for critical data.
+{{< /hint >}}
+
+## Clusters
+
+The **Clusters** screen contains all options related to the cluster feature.
+To see this screen, log in to TrueCommand and click the <span class="iconify" data-icon="mdi:server-network"></span> **Clusters** icon in the upper left.
+
+![ClustersScreen](/images/TrueCommand/2.2/ClustersScreen.png "TrueCommand Clusters Screen")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+When no clusters are created, there is a single option on this screen:
+
+| Setting | Value | Description |
+|---------|-------|-------------|
+| CREATE CLUSTER | button | Opens the form to create a new cluster. |
+
+{{< /expand >}}
+
+### Create Cluster
+
+The cluster creation options are split into two pages: system selection and confirmation.
+
+#### Systems
+
+The initial form has two fields:
+
+![CreateClusterSystems](/images/TrueCommand/2.2/CreateClusterSystems.png "Configuring Systems in the Cluster")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
```

Comment:
suggest removing the third table column for consistency with CORE and SCALE docs
```suggestion
| Setting | Description |
```